config/database.yml.example

```diff
@@ -1,87 +0,0 @@
-
```

Comment:
you need this file

Reply:
Yes, I know, team member has deleted them with a commit..
Fixed.